src/main/resources/assets/sync/lang/fr_fr.json

```diff
@@ -0,0 +1,24 @@
+{
+  "itemGroup.sync.sync": "Sync",
+
+  "block.sync.shell_storage": "Stockage des capsules",
+  "block.sync.shell_constructor": "Fabricateur de capsules",
+  "block.sync.treadmill": "Tapis roulant",
+
+  "item.sync.sync_core": "Noyau de synchronisation",
+
+  "death.attack.sync.fingerstick": "%s est mort pour faire avancer la science !",
```

Comment:
Should an exclamation mark really stand one space apart from the text, or is it just a typo?

Reply:
It is not required, it is just the way I write it, I (or you) can change it !

Reply:
> French. Yes, the whole language. In French, exclamation points, question marks, colons, and semi-colons (all forms of "high" punctuation) should always have a space preceding them. For example, if you wanted to text your friend about how much you hate being online, you'd say Je déteste être en ligne !

Oh, ok. It seems that I don't remember much from my french lessons :D